[Stream] Various Updates (#19980)
cloudflare/cloudflare-docs: src/content/docs/stream/stream-live/watch-live-stream.mdx

import { Render } from "~/components"

When a [Live Input](/stream/stream-live/start-stream-live/) begins receiving a
broadcast, a new video is automatically created if the input's `mode` property
is set to `automatic`.

To watch, Stream offers a built-in Player or you use a custom player with the
HLS and DASH manifests.

<Render file="chromecast_limitations" />

## View by Live Input ID or Video ID

Whether you use the Stream Player or a custom player with a manifest, you can

> reference the Live Input ID or a specific Video ID. The main difference is what

Reviewer: Should we put Live Input ID and Video ID in tick marks? So `Live Input ID` for example?

Author: I actually wanted to chat with you about this and CF Style --- I don't know if my incessant capitalization of Video ID and Live Input ID is appropriate, but putting them in backticks would just be for code, right? 😅

happens when a broadcast concludes.

Use a Live Input ID in instances where a player should always show the active
broadcast, if there is one, or a "Stream has not started" message if the input

> is idle. Best for cases where a page is dedicated to a creator, channel, or

Reviewer: ```suggestion
is idle. This option is best for cases where a page is dedicated to a creator, channel, or
```

recurring program. The Live Input ID is provisioned for you when you create the
input; it will not change.

Use a Video ID in instances where a player should be used to display a single

> broadcast or its recording once the broadcast has conluded. Best for cases where

Reviewer: ```suggestion
broadcast or its recording once the broadcast has concluded. This option is best for cases where
```

a page is dedicated to a one-time event, specific episode/occurance, or date.
There is a _new_ Video ID generated for each broadcast _when it starts._

Using DVR mode, explained below, there are additional considerations.

Stream's URLs are all templatized for easy generation:

**Stream built-in Player URL format:**

```
https://customer-<CODE>.cloudflarestream.com/<INPUT_ID|VIDEO_ID>/iframe
```

A full embed code can be generated in Dash or with the API.

**HLS Manifest URL format:**

```
https://customer-<CODE>.cloudflarestream.com/<INPUT_ID|VIDEO_ID>/manifest/video.m3u8
```

You can also retrieve the embed code or manifest URLs from Dash or the API.

## Use the dashboard

To get the Stream built-in player embed code or HLS Manifest URL for a custom player:

1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.

> 2. Click **Stream** > **Live Inputs**.

Reviewer: ```suggestion
2. Select **Stream** > **Live Inputs**.
```

> 3. Click a live input from the list.

Reviewer: ```suggestion
3. Select a live input from the list.
```

4. Locate the **Embed** and **HLS Manifest URL** beneath the video.

> 5. Determine which option to use and then click **Click to copy** beneath your choice.

Reviewer: ```suggestion
5. Determine which option to use and then select **Click to copy** beneath your choice.
```

> These will reference the Live Input ID.

Reviewer: What is "these" reference to?
